src/cnaas_nms/tools/dhcp_hook.py

```diff
@@ -17,9 +18,18 @@ def get_apidata(configfile='/etc/cnaas-nms/apiclient.yml'):
         return yaml.safe_load(apiclient_file)
 
 
+def get_jwt_token():
+    try:
+        token = os.environ['JWT_AUTH_TOKEN']
```

Comment:
I think this variable needs to be set to something in test/integrationtests.sh for this to work in integration tests